caramel_launcher.sh

```diff
@@ -0,0 +1,12 @@
+#! /bin/sh
+## Launch caramel with pserve as a Pyramid WSGI app using environment variables
```

Comment:
Is this only for development? If so, it should be clearly commented as that, especially as the code here replicates the logic inside the tools and caramel to parse environment variables, but also overrides them with new default values?

Especially since "pserve" is not intended for production usage. 


Our current production launcher is here:
https://gitlab.com/ModioAB/containers/-/blob/master/caramel/launcher

Reply:
Since we are calling pserve I don't see how code in the tools/scripts matter? This never "overrides" anything the tools/scripts use. Currently caramel(as a server) doesn't interact with environment variables as far as I know.